.changeset/lemon-tools-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/calendar": minor
+---
+
+Makes text in the calendar in high contrast mode that is not in a selected cell visible again by removing redundant color property and allowing it to be set by the adjusted variable values.
```

Comment:
By "adjusted variable values," did you mean the `--highcontrast` properties? If so, maybe we could add a set of parentheses to the end of this sentence, just to make it super clear where the color properties are getting set now. For examples, "...to be set by the adjust variable values (`--highcontrast-*` custom properties)."

I'll leave that up to you! 👍